buildnow.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# shellcheck disable=SC2086,SC2162
+
+
+[[ "$1" != "" ]] && BRANCH="$1" || BRANCH="$(git branch --show-current)"
+[[ "$BRANCH" == "main" ]] && TAG="latest" || TAG="$BRANCH"
+[[ "$ARCHS" == "" ]] && ARCHS="linux/armhf,linux/arm64,linux/amd64"
+
+BASETARGET1=ghcr.io/kx1t
+BASETARGET2=kx1t
```

Comment:
should this remain your helper script and not be in the repo? we could pull values like this out of an ignored .env file perhaps.